index.html

```diff
@@ -8343,6 +8343,42 @@ <h1>JSON Schema for TD Instance Validation</h1>
           }
         ]
       },
+      "genericSecurityScheme": {
```

Comment:
This could also be changed to `additionalSecurityScheme`, indicating that it is not one of the pre-defined ones.

Reply:
I changed the name to `additionalSecurityScheme` now as I find it to be a bit clearer.